aspnetcore/signalr/configuration.md

```diff
@@ -1460,6 +1464,7 @@ Additional options can be configured in the `WithUrl` (`withUrl` in JavaScript)
 | JavaScript Option | Default Value | Description |
 | ----------------- | ------------- | ----------- |
 | `accessTokenFactory` | `null` | A function returning a string that is provided as a Bearer authentication token in HTTP requests. |
+| `logMessageContent` | `null` | Set this to `true` to log the bytes/chars of messages being sent and received by the client. |
```

Comment:
Nice catch seeing `logMessageContent` was undocumented. It looks like it's been there since 1.0, so it could be added in the "< aspnetcore-2.2" moniker range.

https://github.com/aspnet/SignalR/blob/e1a22b315dae5fc1335114908a545cd706af6c60/clients/ts/signalr/src/IHttpConnectionOptions.ts#L30-L34

Reply:
Oh whoops, thought I added it to them all